### DISCUSSION TITLE
What is the difference between the livesplit one website and loal host.

### DISCUSSION THREAD

**Discussion by Nisanth-Nimashakavi** [LiveSplit/LiveSplitOne#611] [Help]
also how do i use the sockets feature

**Comment by CryZe** [comment]
There is no difference, they are the same. However it's easier to connect to a websocket server via http rather than https. So I guess in that sense it's easier if you have a local version. You'd need to write a websocket server and then send these commands to control LSO: https://github.com/LiveSplit/LiveSplitOne/blob/83780e6d51ba2190526966a3d686d8105f7e8680/src/ui/TimerView.tsx#L255-L266

**Comment by Nisanth-Nimashakavi** [comment]
is there a tutorial on how to that thank you btw

**Comment by KebabLord** [comment]
@nimnim111 i'd use an existing implementation instead of creating it from scratch
you can directly use this script or use it as a reference
https://github.com/thearst3rd/livesplitone-global-hotkeys
